package.json

```diff
@@ -194,61 +203,58 @@
     "findup": "^0.1.5",
     "fork-ts-checker-webpack-plugin": "^5.1.0",
     "get-port": "^5.1.1",
-    "glob": "^7.1.6",
+    "glob": "^8.1.0",
     "html": "^1.0.0",
-    "html-format": "^1.0.1",
+    "html-format": "^1.0.2",
     "html-webpack-plugin": "^4.4.1",
     "jest": "^24.1.0",
     "jest-cli": "^24.1.0",
-    "moment": "^2.27.0",
-    "moment-timezone": "^0.5.31",
+    "moment": "^2.29.4",
+    "moment-timezone": "^0.5.41",
     "node-sass": "^8.0.0",
     "pegjs": "^0.10.0",
     "postcss-cli": "^7.1.2",
     "postcss-inline-svg": "^4.1.0",
     "postcss-loader": "^4.0.1",
     "pre-commit": "^1.2.2",
     "prettier": "^2.1.2",
-    "prop-types": "^15.6.0",
-    "puppeteer": "^5.5.0",
+    "puppeteer": "^19.11.1",
```

Comment:
@BSFishy  Can you run `test-a11y` on your side and see if it exits correctly? I've seen some issues where `browser.close()` didn't work correctly because there are open tabs that were not closed. If the issue is confirmed, I will roll back to a previous version and create a new PR, where this problem will be solved as follows:
```js
const pages = await browser.pages();
for (let i = 0; i < pages.length; i++) {
  await pages[i].close();
}
await browser.close();
```

Reply:
Tests fail, of course, because #640 hasn't been merged yet. However, beyond that it looks like everything exits properly. However, if there is an issue related to a specific platform (i.e. if this fix is needed for M1/M2 Macs), we should definitely get it in somewhere